### PR TITLE
Add Integrand:::AVERAGE support to LR ASMs

### PR DESCRIPTION
**Pull request by akva2**
Sits on top of #523

### CODE REVIEW

File: src/ASM/LR/ASMu2D.C [hunk 1]
```diff
@@ -1187,6 +1187,31 @@ bool ASMu2D::integrate (Integrand& integrand,
         dXidu[1] = el->vmax() - el->vmin();
       }
 
+      if (integrand.getIntegrandType() & Integrand::AVERAGE)
+      {
+        // --- Compute average value of basis functions over the element -----
```
Comment (kmokstad) [inline src/ASM/LR/ASMu2D.C:1192]: Cosmetic: Blank line here to be consistent with the similar 3D code.

**Review by kmokstad — APPROVED** [review]
This approval for the last commit only, assuming it will be rebased after fixing the #523 issue first.

**Comment by akva2** [comment]
jenkins build this ifem-stokes=191 please

**Comment by akva2** [comment]
jenkins build this with downstreams ifem-stokes=191 please

**Comment by akva2** [comment]
jenkins build this with downstreams ifem-stokes=191 please

**Pull request merged**